app/src/main/java/com/celzero/bravedns/data/AppMode.kt

```diff
@@ -132,6 +155,7 @@ class AppMode internal constructor(private val context: Context,
     }
 
     fun getConnectedDnsObservable(): MutableLiveData<String> {
+        connectedDNS.postValue(persistentState.connectedDnsName)
```

Comment:
`connectedDNS` value is now posted from different place. Sometimes the observer was not getting values, now the value is posted in `init` instead of getObservable.

Reply:
> `connectedDNS` value is now posted from different place. Sometimes the observer was not getting values, now the value is posted...

Exactly! Avoid calling random functions from random places. We spent 2 months untangling the codebase! Why screw it all up immediately?

Reply:
Removed the observable.